android/src/fabric/java/com/reactnativepagerview/PagerViewViewManager.kt

```diff
@@ -164,6 +164,11 @@ class PagerViewViewManager : ViewGroupManager<NestedScrollableHost>(), RNCViewPa
         return
     }
 
+    @ReactProp(name = "useLegacy")
+    override fun setUseLegacy(view: NestedScrollableHost?, value: Boolean?) {
```

Comment:
```suggestion
    override fun setUseLegacy(view: NestedScrollableHost?, value: Boolean) {
```

Otherwise I got 

'setUseLegacy' overrides nothing

Reply:
Done